### PR TITLE
Revert the use of `<filesystem>`... again

### PR DESCRIPTION
**Pull request by ISSOtm**
This restores compatibility with macOS <10.15, while dropping the hack that endeavoured towards that (but ultimately failed).

Unfortunately we lose some convenience, but the change is fairly minor, and I'd like not to drop a platform in the same situation as Win7 (EOL but still largely used because of later breakage) for something that minor.

### CODE REVIEW

File: src/gfx/main.cpp [hunk 1]
```diff
@@ -329,7 +326,6 @@ static std::vector<size_t> readAtFile(std::filesystem::path const &path,
 		} while (c != '\n' && c != EOF); // End if we reached EOL
 	}
 }
-
```
Comment (Rangi42) [inline src/gfx/main.cpp:332]: This blank line should stay.

**Review by Rangi42 — COMMENTED** [review]
Reverting filesystem looks good. I would expect to force-push to get rid of the 0.7.0 release commit though, like you did over the macos hack issue.

**Comment by ISSOtm** [comment]
I don't want to force-push commits beyond "a couple minutes ago", because some people may have pulled in the meantime. (Ideally we'd never, but sometimes I'm too tired to care.)

**Comment by Rangi42** [comment]
Normally I would agree, but that being a release commit in particular feels wrong to keep. (Consider the disagreement of `rgbasm -V` vs the `__RGBDS_*__` constants when checked out to that commit.)

**Review by Rangi42 — APPROVED** [review]

**Pull request merged**